stingray/lightcurve.py

```diff
@@ -285,14 +285,15 @@ def counts(self, value):
     @property
     def counts_err(self):
         counts_err = self._counts_err
-        if counts_err is None and self._countrate_err is not None:
-            counts_err = self._countrate_err * self.dt
-        elif counts_err is None:
+        if counts_err is None:
```

Comment:
The change of logic here seems wrong. If self._countrate_err is not None, you are calculating counts_err twice, right?

Reply:
Yes just realized that. I'll change it right away

Reply:
Hi @theand9 I still don't understand why changing the logic saves us time here :).